lib/BridgeList.php

```diff
@@ -0,0 +1,121 @@
+<?php
+final class BridgeList {
+
+	private static function head() {
```

Comment:
I would probably name this ```getHead``` or something

Reply:
Yes, good point. I'll change all functions to `get*`